.travis.yml

```diff
@@ -2,7 +2,6 @@ language: scala
 sudo: false
 scala:
   - 2.12.6
-  - 2.11.12
```

Comment:
I removed 2.11.12 Scala version, because there is some incompatibilities.. 
```
[error] 'constant' is not a valid choice for '-Xlint'
[error] bad option: '-Ywarn-extra-implicit'
 bad option: '-Ywarn-unused:imports' ...
```

Reply:
Maybe this can be solved in https://github.com/scalaz/scalaz-sbt/issues/21